### ISSUE TITLE
Random GUI bugs/crashes

### ISSUE THREAD

**Issue by MancusNemo** [PadWorld-Entertainment/worldofpadman#327]
Hello,

when I try to load a new map with GUI it didn't work always. Anytime it does, anytime not. But with console it works always.

The Log files are empty. WTF

Anytime maps are missing in the list, anytime all are there. oO

Only once I saw whats going on. It says on the padgarden night map can't load sunshine and then Linux Mint says Process is crashed.
But in this map is no sun!? WTF

But normal there is nothing in the console or the terminal or in the log files. Its normal or empty.

I reinstalled it, it would a bit better. I checked hashes of the zip. There is no mismatch when I loaded the file more than one. All the same. So this can't be the error.

I am using Linux Mint 21.3

**Comment by kai-li-wop** [comment]
Can you please make a video? It is hard to understand from your description.

**Comment by kai-li-wop** [comment]
@MancusNemo: Without any additional information I will close this as won't fix soon.
